TwoPaneLayout/library/src/androidTest/java/com/microsoft/device/dualscreen/twopanelayout/TwoPaneTest.kt

```diff
@@ -33,51 +30,67 @@ import kotlin.math.roundToInt
 @RunWith(AndroidJUnit4::class)
 class TwoPaneTestTest : LayoutTest() {
```

Comment:
It's fine to have duplicate word "Test"?

Reply:
@ancirja-m Good catch! Thanks!